ImageProcessing/README.md

```diff
@@ -0,0 +1,747 @@
+US West (Oregon) | [![Launch Module in us-west-2](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks/new?stackName=wildrydes-step-module-resources&templateURL=https://s3-us-west-2.amazonaws.com/wild-rydes-step-module-us-west-2/0-cfn/wild-rydes-step-module-us-west-2.output.yaml)
+EU (Ireland) | [![Launch Module 1 in eu-west-1](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/images/cloudformation-launch-stack-button.png)](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=wildrydes-step-module-resources&templateURL=https://s3-eu-west-1.amazonaws.com/wild-rydes-step-module-eu-west-1/0-cfn/wild-rydes-step-module-eu-west-1.output.yaml)
+
+<details>
+<summary><strong>AWS CloudFormation Launch Instructions (expand for details)</strong></summary><p>
+
+1. Click the **Launch Stack** link above for the region of your choice.
+
+1. Click **Next** on the Select Template page.
+
+1. On the Specify Details page, leave all the defaults and click **Next**.
+
+1. On the Options page, leave all the defaults and click **Next**.
+
+1. On the Review page, Click the checkboxes to give AWS CloudFormation permission to **"create IAM resources"** and **"create IAM resources with custom names"**
+
+1. Click **"Create Change Set"** in the Transforms section
+
+1. Click **"Execute"**
+
+1. Wait for the `wildrydes-step-module-resources` stack to reach a status of `CREATE_COMPLETE`.
+
+1. With the `wildrydes-step-module-resources` stack selected, click on the **Outputs** tab. These resources will be referenced in the later steps. 
+
+</p></details>
+
+
+
+### 3. Create an initial AWS Step Functions state machine
+
+After the riders upload their photo, the first thing we need do in our processing pipeline is to run a face detection algorithm on it to verify that it has a recognizable face in the photo (zero or multiple faces in the photo doesn't help unicorns recognize the rider) and the face is not wearing sunglasses (makes it harder to identify the rider). If these validations fail, notify the user and end the workflow.
+
+The AWS Lambda function that implements this check by leveraging the **Amazon Rekognition** deep-learning based image analysis API is already deployed by AWS CloudFormation in the previous step. Look in the **Outputs** section for `FaceDetectionFunctionArn` for the ARN of the Lambda function. 
+
+The AWS Lambda function to be called when the validations fail is deployed by the AWS CloudFormation called `NotificationPlaceholderFunction`. The intent behind this step is to notify the user the photo validation failed and the error reason, so they can try upload a different photo. To keep this workshop module short, the function just prepares the message instead of actually sending the message (feel free to modify the code to send email notifications through SES)
```

Comment:
Maybe add this as an Extra Credit item at the very end to reduce the amount of text here?